aeron-driver/src/main/java/io/aeron/driver/media/SendChannelEndpoint.java

```diff
@@ -734,13 +755,20 @@ else if (datagramChannel.isOpen())
                 buffer.position(position);
                 channelEndpoint.sendHook(buffer, destination);
                 bytesSent = datagramChannel.send(buffer, destination);
+
+                if (bytesSent < bytesToSend)
+                {
+                    transportShortSends.incrementRelease();
+                }
             }
         }
         catch (final PortUnreachableException ignore)
         {
+            transportShortSends.incrementRelease();
         }
         catch (final IOException ex)
         {
+            transportShortSends.incrementRelease();
```

Comment:
I'm not sure if we should track this one in the same counter at all.  I think errors are not short sends in this case.  However, we'd need to double check the exact errors we get here.  I think these end up being configuration type errors.